### PR TITLE
improve documentation

### PR DESCRIPTION
**Pull request by patriknw**
before diving into the code I read the documentation and tried the demo :-)

### CODE REVIEW

File: docs/src/main/paradox/bootstrap.md [hunk 1]
```diff
@@ -87,74 +87,60 @@ of them should perform this join. We make this decision by sorting the known add
...
-which we recommend (and implement) with DNS, though you may use alternative lookup methods if you want. The bootstrap 
-process will then start probing those nodes Contact Points (which are HTTP endpoints, exposed via Akka Management by the
-Bootstrap Management Extension) for known seeds to join. Since no cluster exists yet, the seed node replies will be 
-empty from all contact points. Once at least `akka.management.cluster.bootstrap.required-contact-point-nr` nodes are
-discovered and `akka.management.cluster.bootstrap.stable-margin` time has passed, the nodes will decide that it is time 
-to confidently (enough) form a new cluster. The node with the "lowest" contact point address will decide to join itself,
-and other nodes are expecting this awaiting a seed-node address to appear in the contact point probing responses.
-
-Once the lowest addressed node has joined itself, it has formed a new cluster, and therefore starts returning itself
-as seed-node in its Contact Point response. Other nodes notice this and will join it, growing the existing cluster.
```
Comment (patriknw) [inline docs/src/main/paradox/bootstrap.md]: this 3rd repetition of the same thing was too much for my taste, made it more concise hopefully without loosing anything

File: docs/src/main/paradox/bootstrap.md [hunk 2]
```diff
@@ -245,15 +231,7 @@ file for the service configuration, which is:
 
 @@snip [akka-cluster.yml](../../../../bootstrap-joining-demo/kubernetes-api/kubernetes/akka-cluster.yml) 
 
-In order to run the example you can build it via:
-
-```
-sbt shell
-> project bootstrap-joining-demo
-> docker:publishLocal
-```
```
Comment (patriknw) [inline docs/src/main/paradox/bootstrap.md]: This fails if minikube has not been started first, and the same thing is anyway repeated below...

File: docs/src/main/paradox/bootstrap.md [hunk 3]
```diff
@@ -2,12 +2,12 @@
 # Akka Cluster Bootstrap
 
 The bootstrap module / extension allows an Akka Cluster to (semi) automatically discover its neighbouring nodes,
-and join the current node to them if a cluster already exists, or safely form a new cluster for all those nodes,
-if no cluster exists there yet.
+and join the existing cluster, or safely form a new cluster for discovered nodes.
+
+The bootstrap processes may be configured to use various implementations of how to discover other nodes. One
+implementation utilises DNS records and this works particularity well in environments like Kubernetes or Mesos
+where DNS records are managed for Services automatically.
```
Comment (ktoso) [inline docs/src/main/paradox/bootstrap.md]: services (lowercase)?

File: docs/src/main/paradox/bootstrap.md (hunk 1 above)
Comment (ktoso) [reply docs/src/main/paradox/bootstrap.md]: Yes it seems all info is still there, thanks

File: docs/src/main/paradox/bootstrap.md [hunk 4]
```diff
@@ -87,74 +87,60 @@ of them should perform this join. We make this decision by sorting the known add
...
-Once the lowest addressed node has joined itself, it has formed a new cluster, and therefore starts returning itself
-as seed-node in its Contact Point response. Other nodes notice this and will join it, growing the existing cluster.
 
 The illustration below may be of help in visualising this process:
 
 ![project structure](images/bootstrap-forming-cluster.png)
 
-The reason to not use Akka's remoting in the contact point probing itself is to in the future enable upgrades semi-automatic between even not wire compatible binary versions or protocols (e.g. moving from a classic remoting based system to an artery based one), or even more advanced deployment patterns.
+Additionally, a setting is provided, `akka.management.cluster.bootstrap.form-new-cluster` that can be disabled to only allow the
+node to join existing clusters (Case 2 below). This can be used to provide additional safety during the typical deployment
```
Comment (ktoso) [inline docs/src/main/paradox/bootstrap.md]: reads good

File: docs/src/main/paradox/bootstrap.md (hunk 2 above)
Comment (ktoso) [reply docs/src/main/paradox/bootstrap.md]: good catch ,thanks

**Comment by ktoso** [comment]
Nice to have another look at it, thanks!

**Pull request merged**